### PR TITLE
Add: Zoologist and update from Geologist

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
This adds the Zoologist code from wpcom and updates it with the latest changes from Geologist.

### CODE REVIEW

**Comment by pbking** [comment]
Looks groovy to me as far as I can tell.  I'll leave a final sign off to @alaczek

File: zoologist/child-theme.json [hunk 1]
```diff
@@ -164,12 +165,12 @@
 			"fontSizes": [
 				{
 					"name": "Small",
-					"size": "16px",
```
Comment (alaczek) [inline zoologist/child-theme.json]: I made these font-size changes on purpose, to match the design in Figma, so I'd like to keep them in place.

File: zoologist/child-theme.json [hunk 2]
```diff
@@ -164,12 +165,12 @@
...
+					"size": "18px",
 					"slug": "small"
 				},
 				{
 					"name": "Normal",
-					"size": "18px",
```
Comment (alaczek) [inline zoologist/child-theme.json]: I made these font-size changes on purpose, to match the design in Figma, so I'd like to keep them in place.

File: zoologist/child-theme.json [hunk 3]
```diff
@@ -228,15 +229,10 @@
 				}
 			},
 			"core/post-title": {
-				"spacing": {
```
Comment (alaczek) [inline zoologist/child-theme.json]: I made this change on purpose. Zoologist uses Query block on the home page, and some spacing is needed between the post title and the content. 
I'm actually wondering if this is a Geologist issue? Take a look at this site using Geologist where the Home page is set using Query block https://zoologiststarter.wordpress.com/, and compare to Geologist demo https://geologistdemo.wordpress.com/ that uses the "Display posts on front" setting in the Customizer.

File: zoologist/child-theme.json [hunk 4]
```diff
@@ -228,15 +229,10 @@
...
-					"margin": {
-						"bottom": "var(--wp--custom--gap--vertical)"
-					}
-				},
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
 					"fontSize": "min(max(48px, 7vw), 64px)",
-					"fontWeight": 700,
```
Comment (alaczek) [inline zoologist/child-theme.json]: I made this change on purpose, to match the design in Figma, so I'd like to keep it in place.

**Review by alaczek — CHANGES_REQUESTED** [review]
I left some inline code comments - tl;dr is I made some design changes in Zoologist on purpose and I'd like to keep them.

File: blockbase/rebuild-child-themes.sh [hunk 5]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-declare -a ChildThemes=( "mayland-blocks" "seedlet-blocks" "quadrat" "skatepark" "geologist" "videomaker" )
+declare -a ChildThemes=( "mayland-blocks" "seedlet-blocks" "quadrat" "skatepark" "geologist" "videomaker" "zoologist" )
```
Comment (alaczek) [inline blockbase/rebuild-child-themes.sh:3]: Can you explain what this does exactly? :) 
We're planning to generate a few more themes from Blockbase, do I need to add them here as well?

Comment (MaggieCabrera) [reply blockbase/rebuild-child-themes.sh:3]: this allows you to run `npm run build:all` from blockbase's folder, and that builds both the compiled CSS and theme.json files of all the children in the list. That means each time we update blockbase, the changes will propagate to all the children's theme.json files.

File: zoologist/theme-gen-readme.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+and applying small changes to it from what it can see on the starter site.
+
+For example, if I have the blank-canvas theme and
+a https://fredricksonstarter.wordpress.com/ starter site that contains
+a homepage design with font and color settings, I can create a new fredrickson
+theme.
+
+## Commands
+
+To create my-new-theme based on blank-canvas and
```
Comment (scruffian) [inline zoologist/theme-gen-readme.md:14]: Should this say blockbase?

File: zoologist/theme-gen-readme.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+fredricksonstarter.wordpress.com:
+
+```
+wp --require=wp-content/lib/theme-gen/init.php theme-gen fredricksonstarter.wordpress.com blank-canvas my-new-theme
```
Comment (scruffian) [inline zoologist/theme-gen-readme.md:18]: Should this say blockbase?

File: zoologist/child-theme.json (hunk 2 above)
Comment (scruffian) [reply zoologist/child-theme.json]: Done

File: zoologist/child-theme.json (hunk 1 above)
Comment (scruffian) [reply zoologist/child-theme.json]: Done

File: zoologist/child-theme.json (hunk 4 above)
Comment (scruffian) [reply zoologist/child-theme.json]: Done

File: zoologist/child-theme.json (hunk 3 above)
Comment (scruffian) [reply zoologist/child-theme.json]: I'm not convinced this line does anything, it seems to be overridden by other rules:
Screenshot 2021-10-13 at 13 33 19

**Comment by scruffian** [comment]
We should add a social nav too

File: zoologist/child-theme.json (hunk 3 above)
Comment (alaczek) [reply zoologist/child-theme.json]: What site is this screenshot from? On the Zoologist demo I see the spacing being added under post title:

image

Comment (scruffian) [reply zoologist/child-theme.json]: The screenshot is a local site running Gutenberg on trunk. I left the spacing config in the child-theme.json so I think this should be OK to merge?

**Review by MaggieCabrera — APPROVED** [review]
this LGTM!

**Pull request merged**